teams/vsp/teams/insights-analytics/ga-events-data-dictionaries.md

```diff
@@ -742,116 +754,196 @@ _When the orientation of the way search results are presented is changed, either
+| IDme Login Success | 4 | Hit | Integer | When a user successfully logs in via IDme |
+| MHV Login Attempt | 5 | Hit | Integer | When a user attempts to login via MHV |
+| MHV Login Success | 6 | Hit | Integer | When a user successfully logs in via MHV |
+| DSLogon Attempt | 7 | Hit | Integer | When a user attempts to login via DSLogon |
+| DSLogon Success | 8 | Hit | Integer | When a user successfully logs in via DSLogon |
+| Education - Count of Programs Saved | 9 | Hit | Integer | Occurs with a Vet Tec Completion with the edu-0994 form |
+| File Uploads | 10 | Hit | Integer | When a user uploads a file |
+| Disability Calculator Rows | 11 | Hit | Integer | Number of rows being used in the disability rating calculator found on /disability/about-disability-ratings/ |
+| Unused | 12 | Hit | Integer | - |
+| Unused | 13 | Hit | Integer | - |
+| Unused | 14 | Hit | Integer | - |
+| Unused | 15 | Hit | Integer | - |
+| Unused | 16 | Hit | Integer | - |
+| GIBCT - Profile Benefit Estimate | 17 | Hit | Currency (Decimal) |  |
+| Unused | 18 | Hit | Integer | - |
+| Address Validation Needed | 19 | Hit | Integer | This is triggered with a profile navigation action |
+| Address Suggestion Provided | 20 | Hit | Integer | This is triggered with a profile navigation action |
+| Address Suggestion Used | 21 | Hit | Integer | User leverages an address suggestion when submitting information on their profile |
+| Home Address Suggestion Failures | 22 | Hit | Integer | Can occur when there is a profile save failure |
+| Mailing Address Suggestion Failures | 23 | Hit | Integer | Can occur when there is a profile save failure |
+| Address Override | 24 | Hit | Integer | User does not leverage the address suggestion and instead opts to override and use their own input when submitting information on their profile |
+| Screening Tool Total Results Returned | 25 | Hit | Integer | Number of results displayed on the COVID screening tool |
+| Screening Tool Total Time | 26 | Hit | Time | Time to complete the COVID screening tool |
+| BAM - Quantity Ordered | 27 | Hit | Integer | Used on a form event |
+| STEM Scholarship Criteria Met - Post 911 | 28 | Hit | Integer | Used for if the post-911 eligibility criteria on the STEM Scholarship was met. If met then convert true to `{{JS - Generic - Custom Metric Increment}}` and convert false to `{{Generic Undefined}}`.  |
+| STEM Scholarship Criteria Met - STEM or Teaching | 29 | Hit | Integer | Used for if the STEM or Teaching eligibility criteria on the STEM Scholarship was met. If met then convert true to `{{JS - Generic - Custom Metric Increment}}` and convert false to `{{Generic Undefined}}`.  |
+| STEM Scholarship Criteria Met - Benefits Remaining | 30 | Hit | Integer | Months remaining for edu elegibility criteria use - used in the `Product Ineligibility Alerts` tag.  |
+| Disability - Claim for Spouse | 31 | Hit | Integer | Used in a disability form event to indicate if they are filing a claim for their spouse |
```

Comment:
Each of these L929 - 937 are 'yes' 'no' questions in the form that increment by 1 if yes and nothing if answered no